Rakefile

```diff
@@ -3,3 +3,8 @@ require 'rspec/core/rake_task'
 
 RSpec::Core::RakeTask.new(:spec)
 task default: :spec
+
+desc "Open an irb session preloaded with this library"
```

Comment:
Its for creating an irb session which will load the gem so that we can test for its methods and datas.